utils/docker/images/install-pmdk.sh

```diff
@@ -36,20 +36,24 @@
 
 set -e
 
-mkdir pkgs
-cd pkgs
+git clone https://github.com/pmem/pmdk
+cd pmdk
+git checkout da5de60ae251fc0a51d1df9b3f6276ec5e257e1a
```

Comment:
can't we just use latest master?

Reply:
No. Because rebuilding Docker image for unrelated reason could introduce regression(s) and we would never know which PMDK revision we are testing against.